<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="hasilLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>241</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_LoadCitra">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTrasformasi">
    <property name="title">
     <string>Trasformasi</string>
    </property>
    <addaction name="actionDiscreate_1"/>
    <addaction name="actionDiscreate_2"/>
   </widget>
   <widget class="QMenu" name="menuDeteksi_tepi">
    <property name="title">
     <string>Deteksi tepi</string>
    </property>
    <addaction name="actionSobel"/>
    <addaction name="actionPrewitt"/>
    <addaction name="actionRoberts"/>
    <addaction name="actionCanny_2"/>
   </widget>
   <addaction name="menuTrasformasi"/>
   <addaction name="menuDeteksi_tepi"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDiscreate_1">
   <property name="text">
    <string>Discreate 1</string>
   </property>
  </action>
  <action name="actionDiscreate_2">
   <property name="text">
    <string>Discreate 2</string>
   </property>
  </action>
  <action name="actionSobel">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="actionCanny">
   <property name="text">
    <string>Canny</string>
   </property>
  </action>
  <action name="actionPrewitt">
   <property name="text">
    <string>Prewwit</string>
   </property>
  </action>
  <action name="actionRoberts">
   <property name="text">
    <string>Robert</string>
   </property>
  </action>
  <action name="actionCanny_2">
   <property name="text">
    <string>Canny</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
